app/assets/javascripts/components/overview/details.jsx

```diff
@@ -492,72 +492,74 @@ const Details = createReactClass({
     }
 
     const shared = (
```

Comment:
Why are there changes to `details.jsx`?

Reply:
As initially I was rendering the form in the edit mode, I was working with `details.jsx`. But, I had removed that part of code. May be some changes still persist. Will correct it.